File: app/src/main/java/com/vividseats/android/challenge/two/HomeCardsAPI.java
package com.vividseats.android.challenge.two;

import com.vividseats.android.challenge.two.data.HomeCard;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface HomeCardsAPI {

    @POST("home/cards")
    Call<List<HomeCard>> getHomeCards(@Body HomeCardRequest homeCardRequest);
}

File: app/src/main/java/com/vividseats/android/challenge/two/data/remote/api/HomeCardsAPI.kt
package com.vividseats.android.challenge.two.data.remote.api

import com.vividseats.android.challenge.two.HomeCardRequest
import com.vividseats.android.challenge.two.data.local.model.HomeCard

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface HomeCardsAPI {

    @POST("home/cards")
    fun getHomeCards(@Body homeCardRequest: HomeCardRequest): Call<List<HomeCard>>
}
